enterprise/templates/enterprise/grant_data_sharing_permissions.html

```diff
@@ -31,7 +31,11 @@
                         </h2>
                     </div>
                     <span class="text">
-                        ${_("In order to link your account, we need consent to share your course data with {provider}.").format(provider=sso_provider)}
```

Comment:
Was this the only dependency on `sso_provider` that needed to be modified to work with `ec_name`?  Also I'd prefer the more verbose `enterprise_customer_name` vs. `ec_name`.